pages/common/lldb.md

```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
```

Comment:
I followed in the steps of `tldr gdb` here, which also just has

```
gdb

The GNU Debugger.
```

Not sure if there's a better way to describe it? The same title is used at https://lldb.llvm.org/ FWIW.